x-pack/plugins/observability_solution/slo/public/pages/slos/components/common/good_bad_events_chart.tsx

```diff
@@ -78,11 +83,11 @@ export function GoodBadEventsChart({
 
   const barClickHandler = (params: XYChartElementEvent[]) => {
     if (slo?.indicator?.type === 'sli.kql.custom') {
-      const [datanum, eventDetail] = params[0];
+      const [dataenum, eventDetail] = params[0];
```

Comment:
Shouldn't this be datum? 
```suggestion
      const [datum, eventDetail] = params[0];
```